package.json

```diff
@@ -116,6 +115,7 @@
     "eslint-plugin-prettier": "^3.4.0",
     "fs-extra": "^10.0.0",
     "fs-teardown": "^0.3.0",
+    "graphql": "^16.3.0",
```

Comment:
Is it safe to migrate to v16 given the optional range is 15/16? Are there any features that are dropped in v16 that we may accidentally rely upon?

Reply:
Since v16 is the version that MSW has used in the latest release, I think it makes sense to keep it as the devDependency version.

There are two main changes in this PR that affect run-time compatibility:
1. Changing usage of `OperationTypeNode` from enum to string
2. The `parse` function

Changing `OperationTypeNode` should be safe to do, the `enum` in v16 maps to the same strings that were used in v15.

The `parse` function from `graphql` is the only piece of runtime code that is actually used by MSW. It's being used to get the `OperationDefinitionNode` for the GraphQL operation that is intercepted. From what I can tell, the fields that MSW is interested in on `OperationDefinitionNode` are unchanged between v15 and v16.

Based on this, I think it should be safe to list v15 as a dependency. 

It's also worth noting that v15 is, at the time of writing, the most commonly downloaded version of `graphql` on NPM (3+ million downloads a week, compared to ~2 million for v16). It would be good to be backwards compatible with v15, because lots of MSW users probably have that version in their projects.